src/components/SideBarItems/SideBarItems.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
We don't need to import React in modern versions of React. I get a red squiggle every time we do this because the `eslint` plugin is reminding me it's not necessary. Do you have the plugin installed and running?